spec/features/votes/user_votes_on_article_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.